site/src/docs/components/breadcrumb/code.mdx

```diff
@@ -0,0 +1,77 @@
+    <a class="hds-link hds-link--medium hds-breadcrumb__link" href="/path/2ndLevelPath/3rdLevelPath">Informal care</a>
+  </div>
+</nav>
+```
+
+</Playground>
+
+### Packages
+
+| Package       | Included                                                                                        | Storybook link                                                                                                                                               | Source link                                                                                                                                                                                                             |
+| ------------- | ----------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **HDS React** | <div style={{ display: 'flex', gap: 'var(--spacing-3-xs)' }}><IconCheckCircleFill /> Yes </div> | <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-breadcrumb--example">View in Storybook</Link>   | <ExternalLink openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/packages/react/src/components/breadcrumb">View source</ExternalLink> |
+| **HDS Core**  | <div style={{ display: 'flex', gap: 'var(--spacing-3-xs)' }}><IconCheckCircleFill /> Yes </div> | <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/core/?path=/story/components-breadcrumb--breadcrumb">View in Storybook</Link> | <ExternalLink openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/packages/core/src/components/breadcrumb">View source</ExternalLink>  |
+
+### Properties
+
+Note! You can find the full list of properties in the <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-breadcrumb--example">React Storybook.</Link>
+
+| Property | Description                                                               | Values                 | Default value |
+| -------- | ------------------------------------------------------------------------- | ---------------------- | ------------- |
+| `list`   | Array of breadcrumb items. Each item is an object with `title` and `path` | `BreadcrumbListItem[]` | -             |
+| `label`  | `aria-label` of the `<nav>` element created by the component              | `string`               | Murupolku     |
```

Comment:
If the default value is removed, remember the remove it from here too.

Reply:
Done.